installation/install.sh

```
@@ -2086,6 +2086,18 @@ uz_upgrade_tomcat(){
         fi
     fi
 
+    # Handle webapp migration when Tomcat version is the same
+    if [ "$TOMCAT_NAME_OLD" = "$TOMCAT_NAME_NEW" ]; then
+        if [ "$OLD_WEBAPP_NAME" != "cloud" ] \&\& [ -d "$TOMCAT_PATH/$TOMCAT_NAME_OLD/webapps/$OLD_WEBAPP_NAME" ]; then
```

Shellcheck (0.11.0)

[error] Couldn't parse this if expression. Fix to allow more checks.

(SC1073)